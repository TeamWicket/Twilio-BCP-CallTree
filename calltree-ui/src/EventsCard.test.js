import EventsCard from './EventsCard';
import React from 'react';
import { shallow } from 'enzyme';

describe("Event Card", () => {
    it("should match snapshot of Events Card Component", () => {
        expect(shallow(<EventsCard />)).toMatchSnapshot();
    })
})
