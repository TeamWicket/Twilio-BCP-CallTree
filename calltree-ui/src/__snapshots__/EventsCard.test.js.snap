// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event Card should match snapshot of Events Card Component 1`] = `ShallowWrapper {}`;
